### PR TITLE
Add Tertiary background to Card and Panel

### PR DESCRIPTION
**Pull request by Vineet119**
#3452

### CODE REVIEW

File: packages/theme/css/palette/neutral.css [hunk 1]
```diff
@@ -40,4 +44,8 @@
   --salt-palette-neutral-secondary-separator: var(--salt-color-white-fade-separatorOpacity-secondary);
   --salt-palette-neutral-tertiary-separator: var(--salt-color-white-fade-separatorOpacity-tertiary);
   --salt-palette-neutral-selection: var(--salt-color-white-fade-background-selection);
+  --salt-palette-neutral-tertiary-background: var(--salt-color-gray-500);
+  --salt-palette-neutral-tertiary-background-disabled: var(--salt-color-gray-500-fade-background);
+  --salt-palette-neutral-tertiary-border: var(--salt-color-gray-300);
+  --salt-palette-neutral-tertiary-border-disabled: var(--salt-color-gray-300fade-background);
```
Comment (joshwooding) [inline packages/theme/css/palette/neutral.css:50]:
```suggestion
  --salt-palette-neutral-tertiary-border-disabled: var(--salt-color-gray-300-fade-background);
```

File: site/docs/components/card/examples.mdx [hunk 2]
```diff
@@ -20,7 +20,7 @@ By default, the card component does not have any interactive or focusable elemen
 <LivePreview componentName="card" exampleName="Variants" >
 ## Variants
 
-All card components provide both primary and secondary variants, offering flexibility and enabling differentiation based on visual hierarchy within your UI.
+All card components provide both primary, secondary and tertiary variants, offering flexibility and enabling differentiation based on visual hierarchy within your UI.
```
Comment (joshwooding) [inline site/docs/components/card/examples.mdx:23]:
```suggestion
All card components provide primary, secondary and tertiary variants, offering flexibility and enabling differentiation based on visual hierarchy within your UI.
```

File: site/docs/components/panel/examples.mdx [hunk 3]
```diff
@@ -25,4 +25,12 @@ Use the secondary variant as the background for less important content.
 
 </LivePreview>
 
+<LivePreview componentName="panel" exampleName="Tertiary" >
+
+## Tertiary
+
+Use the Tertiary variant as the background for sub content.
```
Comment (joshwooding) [inline site/docs/components/panel/examples.mdx:32]: We need to define "sub content"

**Review by joshwooding — CHANGES_REQUESTED** [review]
Needs a changeset

**Comment by joshwooding** [comment]
Will LinkCard and InteractableCard be looked at too?

File: site/docs/components/panel/examples.mdx (hunk 3 above)
Comment (Vineet119) [reply site/docs/components/panel/examples.mdx:32]: done

File: site/docs/components/card/examples.mdx (hunk 2 above)
Comment (Vineet119) [reply site/docs/components/card/examples.mdx:23]: done

File: packages/theme/css/palette/neutral.css (hunk 1 above)
Comment (Vineet119) [reply packages/theme/css/palette/neutral.css]: done

**Comment by Vineet119** [comment]
> Will LinkCard and InteractableCard be looked at too?

the scope for this is only for Card and Panel as per the https://github.com/jpmorganchase/salt-ds/issues/3452, let me know if scope needs to be changed
cc: @origami-z

**Comment by origami-z** [comment]
> > Will LinkCard and InteractableCard be looked at too?
> 
> the scope for this is only for Card and Panel as per the #3452, let me know if scope needs to be changed cc: @origami-z

Let's add those in as well. They are considered part of Card from public facing examples - https://www.saltdesignsystem.com/salt/components/card/examples

Sorry for the confusion

**Comment by Vineet119** [comment]
> > > Will LinkCard and InteractableCard be looked at too?
> > 
> > 
> > the scope for this is only for Card and Panel as per the #3452, let me know if scope needs to be changed cc: @origami-z
> 
> Let's add those in as well. They are considered part of Card from public facing examples - https://www.saltdesignsystem.com/salt/components/card/examples
> 
> Sorry for the confusion

added

File: packages/theme/css/characteristics/container-next.css
Comment (origami-z) [inline packages/theme/css/characteristics/container-next.css:1]: Will need another changeset on `@salt-ds/theme`

Comment (Vineet119) [reply packages/theme/css/characteristics/container-next.css:1]: added

File: .changeset/slimy-dragons-promise.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+"@salt-ds/theme": minor
+---
+
+Added tertiary variables in theme next and existing theme
```
Comment (origami-z) [inline .changeset/slimy-dragons-promise.md:5]: Some token only exist in theme next, some are shared, better to split.

Also need to mention un-deprecate

**Review by joshwooding — DISMISSED** [review]
@origami-z's feedback needs to be addressed

File: .changeset/slimy-dragons-promise.md (hunk 4 above)
Comment (Vineet119) [reply .changeset/slimy-dragons-promise.md]: done

**Review by origami-z — DISMISSED** [review]

**Review by joshwooding — DISMISSED** [review]

File: packages/core/src/interactable-card/InteractableCard.css [hunk 5]
```diff
@@ -24,6 +24,13 @@
   --card-accent-color: var(--salt-container-secondary-borderColor);
 }
 
+/* Styles applied to InteractableCard when variant="tertiary" */
+.saltInteractableCard-tertiary {
+  background: var(--saltInteractableCard-background, var(--salt-container-tertiary-background));
+  border-color: var(--salt-container-tertiary-border);
```
Comment (joshwooding) [inline packages/core/src/interactable-card/InteractableCard.css:30]:
```suggestion
  border-color: var(--salt-container-tertiary-borderColor);
```

File: packages/core/src/interactable-card/InteractableCard.css [hunk 6]
```diff
@@ -24,6 +24,13 @@
...
+  --card-accent-color: var(--salt-container-tertiary-border);
```
Comment (joshwooding) [inline packages/core/src/interactable-card/InteractableCard.css:31]:
```suggestion
  --card-accent-color: var(--salt-container-tertiary-borderColor);
```

File: packages/core/src/link-card/LinkCard.css [hunk 7]
```diff
@@ -24,6 +24,13 @@
   --linkCard-accent-color: var(--salt-container-secondary-borderColor);
 }
 
+/* Styles applied to LinkCard when variant="tertiary" */
+.saltLinkCard-tertiary {
+  background: var(--saltLinkCard-background, var(--salt-container-tertiary-background));
+  border-color: var(--salt-container-tertiary-border);
```
Comment (joshwooding) [inline packages/core/src/link-card/LinkCard.css:30]:
```suggestion
  border-color: var(--salt-container-tertiary-borderColor);
```

File: packages/core/src/link-card/LinkCard.css [hunk 8]
```diff
@@ -24,6 +24,13 @@
...
+  --linkCard-accent-color: var(--salt-container-tertiary-border);
```
Comment (joshwooding) [inline packages/core/src/link-card/LinkCard.css:31]:
```suggestion
  --linkCard-accent-color: var(--salt-container-tertiary-borderColor);
```

**Review by joshwooding — APPROVED** [review]

**Review by dplsek — APPROVED** [review]

**Pull request merged**